### DISCUSSION TITLE
Project is back to life?

### DISCUSSION THREAD

**Discussion by v3ss0n** [Semantic-Org/Semantic-UI#7102] [General]
I thought it was succeed by formaticUI ?

**Comment by rupali-codes** [comment]
Yes, so many bugs getting resolved

**Reply by Adamcina** [reply]
Yes what, is it getting back to life or is it succeeded by FUI? :)

**Comment by rittschapp** [comment]
What is happening with this project? From https://semantic-ui.com/introduction/getting-started.html the installation instructions don't work as the code is not compatible with the current gulp implementation (Error [ERR_REQUIRE_ESM]: require() of ES Module).

And indeed gulp itself looks pretty much dead.

**Reply by v3ss0n** [reply]
Just check formatic-ui for now
